packages/core/test/unit/markdown-it-custom-component.test.js

```diff
@@ -0,0 +1,96 @@
+const markdownIt = require('../../src/lib/markdown-it');
+const { injectTags } = require('../../src/lib/markdown-it/patches/custom-component/customComponentPlugin');
+
+test('markdown-it should parse minimized panel as inline element and normal panel as block element', () => {
+  injectTags([]); // init custom component plugin
+
+  const initSource = (tag) => {
+    const source = [
+      'markdown pre text **immediately** prepend a component, without an empty line after',
+      tag,
+      'line 1 which behaves as per normal ( empty line still counts )\n',
+      'lines 3 onward are placed outside of the component',
+      '...',
+      '...',
+      '</panel>',
+    ];
+
+    return source.join('\n');
+  };
+
+  const tag1 = '<panel minimized>'; // should be rendered as inline element
+  const tag2 = '<panel alt="hi" header="hi" minimized>'; // should be rendered as inline element
+  const tag3 = '<panel alt="hi" minimized header="hi">'; // should be rendered as inline element
+  const tag4 = '<panel minimized alt="hi" header="hi">'; // should be rendered as inline element
+  const tag5 = '<panel alt="hi" header="hi">'; // should be rendered as block element
```

Comment:
would this one work? - `<panel header="test for minimized as part of attribute value">`


I was actually just expecting functional site tests here since they're easier to create/maintain and directly test e2e behaviour 😅, but this is great as well.

But _if you're up for it_, let's move it over somewhere to `functional/test_site`.

Reply:
> would this one work? - `<panel header="test for minimized as part of attribute value">`

Yep! The behaviour is as expected. 

> I was actually just expecting functional site tests here since they're easier to create/maintain and directly test e2e behaviour 😅, but this is great as well.
> 
> But _if you're up for it_, let's move it over somewhere to `functional/test_site`.

No problem! Have moved the tests over to `functional/test_site` :-)